easybuild/easyconfigs/p/PyQt/PyQt-4.11.4-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'PyQt'
+version = '4.11.4'
+
+homepage = 'http://www.riverbankcomputing.co.uk/software/pyqt'
+description = """PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(name)s-x11-gpl-%(version)s.tar.gz']
+source_urls = ['http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-%(version)s']
+
+python = 'Python'
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
```

Comment:
@ysagon please use `%(pyver)s`, and move up `versionsuffix = ...` right below `version = ...`

```python
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
...
```

and then drop the `python`, `pyver` and `pyshortver` local variables entirely...